docs/howto/features/index.md

```diff
@@ -5,85 +5,24 @@ There are several extra features and services that can be enabled on the 2i2c Ju
 These add extra functionality, connect with cloud services, etc.
 See the sections below for more details.
 
-## Layers
```

Comment:
I really appreciate this, thank you. This page has been bothering me - mainly because it's called "enable specific hub features" and the first entry was "deploy a kubernetes cluster" 🙄